libethereum/ExtVM.cpp

```diff
@@ -35,13 +35,20 @@ static unsigned const c_depthLimit = 1024;
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
 static size_t const c_singleExecutionStackSize =
 #ifdef NDEBUG
-	12 * 1024;
+	10 * 1024;
 #else
-	33 * 1024;
+	16 * 1024;
```

Comment:
They don't need to. But they can. I would like to keep it quite low to detect issues as early as possible and to keep offloading point as high as possible.